src/Host/Broker/Impl/Interpreters/InterpreterManager.cs

```diff
@@ -19,6 +19,7 @@ public class InterpreterManager {
         private readonly ROptions _options;
         private readonly ILogger _logger;
         private IFileSystem _fs;
+        private int _intepretedId = 1;
```

Comment:
`_nextInterpreterId`?

But given that `Initialize` is only supposed to be called once, and overwriters `Interpreters` entirely, it could be a local variable inside that method.